deploy/helm/kuberhealthy/values.yaml

```
@@ -2,6 +2,10 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+checkReaper:
+  checkCleanupDuration: 15m # the maximum age of check pods before being reaped
```

can we rename this? not accurate description since this only applies to khjobs not checks... 
```suggestion
  jobCleanupDuration: 15m # the maximum age of kh job pods before being reaped
```